app/views/miq_request/_request_dialog_details.html.haml

```diff
@@ -28,7 +28,25 @@
       = h(field.values.detect { |k, _v| k == wf.value(field.name) }.try(:last) || wf.value(field.name))
 
     - when "DialogFieldDropDownList"
-      = h(field.value.blank? ? _("<None>") : field.value.to_s.gsub("\x1F", ", "))
+      - if field.value.class == Integer
```

Comment:
Hey @GilbertCherrie , do you remember why this change was made?

Reply:
No I don't